Circle.elm

```diff
@@ -30,7 +30,7 @@ main =
 view : Float -> Html msg
 view t =
     WebGL.toHtml
-        [ width 400
+        [ width 800
```

Comment:
Is this change really needed?

Reply:
No, just pushed. I had started playing around, sorry :dancer: